### DISCUSSION TITLE
Pie Charts with Text

### DISCUSSION THREAD

**Discussion by nick-trustlab** [vega/altair#2589] [General]
Adding text to pie charts is not documented but I was able to do this in Altair by adding the "stack" variable.  I think the pie chart example could be improved to include this.  For example:

```python
base = alt.Chart(plot_df).mark_arc(outerRadius=80).encode(
    theta=alt.Theta('count:Q'),
    color=alt.Color('Region:N'),
)

cat_text = alt.Chart(plot_df).mark_text(radius=100).encode(
    theta=alt.Theta('count:Q', stack=True),
    text='Region'
)

base + cat_text
```

**Comment by jakevdp** [comment]
Thanks! we have a similar example in the documentation here: https://altair-viz.github.io/gallery/radial_chart.html

Is there some way we could make this example more discoverable?
